components/admin_console/system_roles/system_role/system_role.test.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import {TestHelper} from 'utils/test_helper';
+
+import SystemRole from './system_role';
+
+describe('admin_console/system_role', () => {
+    const props = {
+        role: TestHelper.getRoleMock(),
+        isDisabled: false,
+        isLicensedForCloud: false,
+        actions: {
+            editRole: jest.fn(),
+            updateUserRoles: jest.fn(),
+            setNavigationBlocked: jest.fn(),
+        },
+    };
+
+    test('should match snapshot', () => {
+        const wrapper = shallow(
+            <SystemRole
+                {...props}
+            />);
+
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should match snapshot with isLicnesedForCloud = true', () => {
```

Comment:
```suggestion
    test('should match snapshot with isLicensedForCloud = true', () => {
```

Reply:
Haha oh wow that is embarrassing! 😅 Thanks for catching these!